databases/src/connect/DatabaseConnectionCreateStep.ts

```diff
@@ -0,0 +1,65 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as url from "url";
+import { Progress } from "vscode";
+import { AzureWizardExecuteStep } from "vscode-azureextensionui";
+import { ext } from "../extensionVariables";
+import { createCosmosDBClient } from "../utils/azureClients";
+import { localize } from "../utils/localize";
+import { nonNullProp } from "../utils/nonNull";
+import { IConnectDBWizardContext } from "./IConnectDBWizardContext";
+import * as azureUtils from "../utils/azureUtils";
+
+
+export class DatabaseConnectionCreateStep extends AzureWizardExecuteStep<IConnectDBWizardContext> {
+    public priority: number = 200;
+    public async execute(context: IConnectDBWizardContext, progress: Progress<{ message?: string; increment?: number }>): Promise<void> {
+        const creatingMessage: string = localize('creatingDatabasesConnection', 'Creating Databases connection...');
+        ext.outputChannel.appendLog(creatingMessage);
+        progress.report({ message: creatingMessage });
+        if (context.server) {
+            const hostName = nonNullProp(context.server, 'name');
+            const port = '5432';
```

Comment:
It'd be good to extract `port` to a constant since we use this in a couple of files.